code/modules/mob/living/living_defense.dm

```diff
@@ -73,10 +73,9 @@
 		return shock_damage
 
 /mob/living/emp_act(severity)
-	var/list/L = src.get_contents()
```

Comment:
and this is why that `istype(loc, /mob/living)` was needed; it fetches all contents, which is a super snowflakey proc to begin with--as such, you have to explicitly code exceptions just to prevent it from EMP'ing things twice....so dumb.